### PR TITLE
[V2]Add method which can be use for restore for interface file from backup

### PR DESCRIPTION
**Pull request by PraveenPenguin**
As we need to revert back the interface file when test completed
so added utility can be used for same

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

File: avocado/utils/network/interfaces.py [hunk 1]
```diff
@@ -313,3 +313,28 @@ def remove_ipaddr(self, ipaddr, netmask):
         except Exception as ex:
             msg = 'Failed to remove ipaddr. {}'.format(ex)
             raise NWException(msg)
+
+    def restore_from_backup(self):
+        """Revert interface file from backup
```
Comment (beraldoleal) [inline avocado/utils/network/interfaces.py:318]: Missing dot.

File: avocado/utils/network/interfaces.py [hunk 2]
```diff
@@ -313,3 +313,28 @@ def remove_ipaddr(self, ipaddr, netmask):
...
+        elif current_distro.name == 'SuSE':
+            path = "/etc/sysconfig/network"
+        else:
+            msg = 'Distro not supported by API. Could not restore the backup file.'
+            raise NWException(msg)
+        filename = "{}/{}".format(path, filename)
+        backup_file = "{}.backup".format(filename)
+        if os.path.exists(backup_file):
+            shutil.move(backup_file, filename)
+
```
Comment (beraldoleal) [inline avocado/utils/network/interfaces.py:337]: Empty line.

File: avocado/utils/network/interfaces.py (hunk 1 above)
Comment (PraveenPenguin) [reply avocado/utils/network/interfaces.py:318]: ah sure

File: avocado/utils/network/interfaces.py (hunk 2 above)
Comment (PraveenPenguin) [reply avocado/utils/network/interfaces.py:337]: sure

**Pull request merged**